minifi/minifi-c2/minifi-c2-assembly/src/main/resources/conf/authorizations.yaml

```diff
@@ -37,3 +37,31 @@ Paths:
     # Default authorization lets anonymous pull any config.  Remove below to change that.
     - Authorization: ROLE_ANONYMOUS
       Action: allow
+
+  /c2/config/heartbeat:
```

Comment:
What do you think about not placing heartbeat and acknowledge under config. Seems like a higher level concept could be either directly under c2. What do you think?

Reply:
no need to do that, you are right we will need to have distinction between the two implementation so this makes sense.